languages.toml

```
@@ -1380,3 +1380,16 @@ indent = { tab-width = 2, unit = "  " }
 [[grammar]]
 name = "scheme"
 source = { git = "https://github.com/6cdh/tree-sitter-scheme", rev = "27fb77db05f890c2823b4bd751c6420378df146b" }
+
+[[language]]
+name = "v"
+scope = "source.v"
+file-types = ["v", "vv"]
+shebangs = ["v run"]
+roots = []
```

```suggestion
roots = ["v.mod"]
language-server = { command = "vls", args = [] }
auto-format = true
```

This should set up the language server but I haven't tested it. You would need to install `vls` separately, I think the repo has instructions on that: https://github.com/vlang/vls

---

yes it does!